src/components/navigation/TheUserMenu.vue

```diff
@@ -12,10 +25,13 @@
             <v-icon size="20">fa-user-circle</v-icon>
           </v-btn>
         </template>
-        <span>Sign in / Sign out</span>
+        <span>User profile</span>
       </v-tooltip>
     </template>
     <v-list>
+      <v-list-item @click="logout">
```

Comment:
rename onClickHandler

Reply:
I think it's better for the action to be descriptive, rather than a generic onClickHandler. How about onClickLogout?